mediapicker/src/main/java/iam/thevoid/mediapicker/rxmediapicker/RxMediaPicker.java

```diff
@@ -71,14 +74,16 @@
     // Seconds
     private long videoMaxDuration;
 
+    // Int
+    private long videoQuality;
 
-    private static RxMediaPicker instanse;
```

Comment:
sorry :)

Reply:
No prob

Reply:
I'm think to update it for use androidx references instead. But I need to test on some devices until